src/features/import/l10n/messageIds.ts

```diff
@@ -0,0 +1,30 @@
+import { m, makeMessages } from 'core/i18n';
+
+export default makeMessages('feat.import', {
+  back: m('Back'),
+  configuration: {
+    title: m('Import people'),
+  },
+  done: m('Done'),
+  import: m('Import'),
+  restart: m('Restart'),
+  steps: {
+    configure: m('Configure'),
+    import: m('Import'),
+    upload: m('Upload'),
+    validate: m('Validate'),
+  },
```

Comment:
I feel like these belong in some sort of sub-object. A message called `feat.import.back` seems one that should be relevant to the entire import feature, but I'm not sure it is, and I'm not sure it's the only occurrence of a variation of the term "back" anywhere in the import feature.

Reply:
@richardolsson it's not clear for me what is the change that you are suggesting here? I've added my strings inside a sub-object called "uploadDialog" so maybe this is something to fix in the import task?

Reply:
Your PR adds 30 lines (the entire file). Is not all of it your code?

Reply:
No, I merged this part into my branch to avoid conflicts so the strings that are not part of `uploadDialog `are not used by my component. Should I remove them for my PR?

Reply:
I think the different steps should have different sub-objects within this object of message IDs. Slack me if you're not sure what that means.